Packs/IBMMaaS360Security/README.md

```diff
@@ -12,10 +12,12 @@ This pack includes Cortex XSIAM content.
 
 ## What does this pack do?
 
-The pack includes the **IBM MaaS360 Security** integration that allows you to collect Admin Login reports and Admin change audits from IBM MaaS360 Security.
```

Comment:
The pack includes the **IBM MaaS360 Security** integration that allows you to collect Admin Login reports and Admin Change audits from IBM MaaS360 Security.